dialog/custom.go

```diff
@@ -23,12 +23,35 @@ func NewCustom(title, dismiss string, content fyne.CanvasObject, parent fyne.Win
 	return d
 }
 
+// ShowCustom shows a dialog over the specified application using custom
+// content. The button will have the dismiss text set.
+// The MinSize() of the CanvasObject passed will be used to set the size of the window.
+func ShowCustom(title, dismiss string, content fyne.CanvasObject, parent fyne.Window) {
+	NewCustom(title, dismiss, content, parent).Show()
+}
+
+// CustomConfirm is a custom dialog with dismiss and confirm buttons.
+//
+// Since 2.4
+type CustomConfirm struct {
+	*dialog
+
+	confirm *widget.Button
```

Comment:
Wouldn't it be possible to inherit from the ConfirmDialog here instead of duplicating things?

Reply:
Should be resolved now